released/packages/coq-coqprime/coq-coqprime.1.0.6/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "thery@sophia.inria.fr"
+homepage: "http://coqprime.gforge.inria.fr/"
+bug-reports: "http://coqprime.gforge.inria.fr/"
```

Comment:
```suggestion
homepage: "https://thery.gitlabpages.inria.fr/coqprime/"
bug-reports: "https://github.com/thery/coqprime/issues"
```

Reply:
I prefer not to refer to gitlab

Reply:
@thery but this means that the website link will be dead in a few months, as per #1298. What do you want the website for the project to be?

Reply:
The content on the gitlab site is an outdated version of the README.md on gitlhub.